src/com/company/Main.java

```diff
@@ -7,15 +7,15 @@
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         int number = scanner.nextInt();
-        switch ((number + 45) / 45) {
+        switch ((number + 100) / 100) {
             case 1:
-                System.out.println("number is in range 0-45");
+                System.out.println("number is in range 0-100");
```

Comment:
Please add spaces  and arrow
example `number is in range 0 -> 45`

Reply:
done